integration-tests/fixtures/stories-formatted.js

```diff
@@ -0,0 +1,29 @@
+module.exports = stories = [
```

Comment:
no change needed: if the module itself does not use/ref `stories` I think you can just write `module.exports = [ ...` but I need to review